projects/standard-rulesets/src/breaking-changes/helpers/rangeChange.ts

```diff
@@ -0,0 +1,100 @@
+import { OpenApi3SchemaFact } from '@useoptic/openapi-utilities';
+import { RuleError } from '@useoptic/rulesets-base';
+
+export function isNumericProperty(
+  before: OpenApi3SchemaFact,
+  after: OpenApi3SchemaFact
+) {
+  const numericTypes = ['integer', 'number'];
+  if (typeof before.type === 'string' && typeof after.type === 'string') {
+    return (
+      numericTypes.includes(before.type) && numericTypes.includes(before.type)
+    );
+  }
+  return false;
+}
+
+export function numericMaximumRangeShouldNotExpand(
+  before: OpenApi3SchemaFact,
+  after: OpenApi3SchemaFact
+) {
+  if (isNumericProperty(before, after)) {
+    if (
+      typeof before.maximum !== 'undefined' &&
+      typeof after.maximum !== 'undefined'
+    ) {
+      if (before.maximum < after.maximum) {
+        throw new RuleError({
+          message: `range maximum increased: ${before.maximum} -> ${after.maximum}`,
```

Comment:
this looks fine to me for max range increase. Is that a breaking change for clients if they were to receive more than they are expecting? As a server I would be fine with it but I belive a client might not

Reply:
Error messages are clear

Reply:
@savage-alex so it sounds like we need to branch the logic here depending on whether the range is in the request or response?

Reply:
@savage-alex so it sounds like we need to branch the logic here depending on whether the range is in the request or response?

Reply:
I think so. Depending on the direction it would hurt the server or the client. Even if we called it out to let someone know they can then think about whether its desirable or compatible or not.

Reply:
@savage-alex  -- I talked to some other teams and I think the expected behavior here for request / response is a bit different depending on the team. I updated the code, and they currently both will run, but if you want to turn one of them off you could exempt the rules by name `prevent response property numeric range changes` or  `prevent request property numeric range changes`